Source/Logger.swift

```diff
@@ -50,6 +50,9 @@ open class Logger {
 
    // MARK: - Properties
 
+    /// A logger that does not output any messages to writers.
+    public static let disabled: Logger = NoOpLogger()
```

Comment:
This is an awesome change, but it's not backwards compatible. Are you thinking this should go out as Willow 6? If yes, then I'd recommend that we deprecate the optional APIs for now and add the disabled Logger. Then we commit this into a `willow6` branch or something for when we're ready to lump more breaking changes into it. Thoughts?

Reply:
If we are considering a major version bump, there are other things I'd like us to consider. The Writer API feels clunky and I would want to maybe take another look at how `os_log` and `os_signpost` work these days and see if there is anything for us to learn from. I also think the default ConsoleWriter could have cleaner output... it's the first thing I replace every time I use Willow...